tools/versioner/cmd/root.go

```diff
@@ -76,243 +61,68 @@ func Execute() {
-	// don't write a changelog for a new module
-	if mod.NewModule() {
 		return nil
-	}
-	const changeLogName = "CHANGELOG.md"
-	rpt := mod.GenerateReport()
-	log, err := os.Create(filepath.Join(stage, changeLogName))
-	if err != nil {
-		return fmt.Errorf("failed to create %s: %v", changeLogName, err)
-	}
-	defer log.Close()
-	if rpt.IsEmpty() {
-		_, err = log.WriteString("No changes to exported content compared to the previous release.\n")
-		return err
-	}
-	_, err = log.WriteString(rpt.ToMarkdown())
-	return err
+	})
+	return stages, err
 }
 
-// returns a slice of tags for the specified repo and tag prefix
-func getTags(repoPath, tagPrefix string) ([]string, error) {
-	wt, err := repo.Get(repoPath)
-	if err != nil {
-		return nil, err
-	}
-	return wt.ListTags(tagPrefix + "*")
-}
-
-// returns the tag prefix for the specified package.
-// assumes repo root of github.com/Azure/azure-sdk-for-go/
-func getTagPrefix(pkgDir string) (string, error) {
-	// e.g. /work/src/github.com/Azure/azure-sdk-for-go/services/redis/mgmt/2018-03-01/redis/v2
-	// would return services/redis/mgmt/2018-03-01/redis/v2.0.0
-	repoRoot := filepath.Join("github.com", "Azure", "azure-sdk-for-go")
-	i := strings.Index(pkgDir, repoRoot)
-	if i < 0 {
-		return "", fmt.Errorf("didn't find '%s' in '%s'", repoRoot, pkgDir)
-	}
-	return strings.Replace(pkgDir[i+len(repoRoot)+1:], "\\", "/", -1), nil
-}
-
-// returns the appropriate module tag based on the package version info
-// tags - list of all current tags for the module
-func calculateModuleTag(tags []string, mod modinfo.Provider) (string, error) {
-	if mod.BreakingChanges() && !mod.VersionSuffix() {
-		return "", errors.New("package has breaking changes but directory has no version suffix")
-	}
-	tagPrefix, err := getTagPrefix(mod.DestDir())
-	if err != nil {
-		return "", err
-	}
-	// if this has breaking changes then it's simply the prefix as a new major version
-	if mod.BreakingChanges() {
-		return tagPrefix + ".0.0", nil
-	}
-	if len(tags) == 0 {
-		if mod.VersionSuffix() {
-			panic("module contains a version suffix but no tags were found")
-		}
-		// this is the first module version
-		return tagPrefix + "/" + startingModVer, nil
-	}
-	if !mod.VersionSuffix() {
-		tagPrefix = tagPrefix + "/v1"
-	}
-	tag := tags[len(tags)-1]
-	v := semverRegex.FindString(tag)
-	if v == "" {
-		return "", fmt.Errorf("didn't find semver in tag '%s'", tag)
-	}
-	sv, err := semver.NewVersion(v)
-	if err != nil {
-		return "", fmt.Errorf("failed to parse semver: %v", err)
-	}
-	// for non-breaking changes determine if this is a minor or patch update.
-	if mod.NewExports() {
-		// new exports, this is a minor update so bump minor version
-		n := sv.IncMinor()
-		sv = &n
-	} else {
-		// no new exports, this is a patch update
-		n := sv.IncPatch()
-		sv = &n
-	}
-	return strings.Replace(tag, v, "v"+sv.String(), 1), nil
-}
+//
+//// does the actual work
+//func theCommandImpl(args []string) (string, error) {
```

Comment:
Why is all this commented out?

Reply:
I moved this piece of code to unstage.go. Comments will be removed.